nss-day-cohort-46/sprinkles-of-joy-admiral-ackbar-pudding
# Pp orderhistory order

# Description

added soft delete for orders and sequential order for orderList

## Type of change

- [] New feature (non-breaking change which adds functionality)

# Testing Instructions

```
git fetch --all
git checkout pp-orderhistory-order
```
new orders should be on top of order history
deleting order should not remove it from the api, but change the isDeleted value to true

# Checklist:

- [] I have commented my code, particularly in hard-to-understand areas
